framework/include/interfaces/FunctorInterface.h

```diff
@@ -63,9 +63,27 @@ class FunctorInterface
   const Moose::Functor<T> & getFunctor(const std::string & name, SubProblem & subproblem);
 
   /**
-   * Checks the subproblem for the given functor. This will not query default functors potentially
-   * stored in this object, e.g. this method will return false if the user passed an int or real to
-   * the functor param in the input file
+   * Retrieves a functor from the passed-in subproblem. This method also leverages the ability to
+   * create default functors if the user passed an integer or real in the input file
+   * @param name The actual name of the functor to retrieve instead of the parameter name
```

Comment:
add param for the subproblem

it s probably missing in the equivalent getFunctor routine